tests/import/builtin_import.py

```diff
@@ -9,6 +9,12 @@
 except TypeError:
     print('TypeError')
 
+# module name should not be empty
+try:
+    print(__import__("")) # this breaks if __import__("") returns MP_OBJ_NULL
```

Comment:
The print is not needed, the test will still catch the failure and success without the print.

Reply:
I removed the print.